configs/trainer/default.yaml

```diff
@@ -10,3 +10,13 @@ max_epochs: 10
 
 # ckpt path
 resume_from_checkpoint: null
+
+# Activation learning rate finder
+# auto_lr_find=True
+
+# Precision default:32
+# precision: 32
```

Comment:
there are dozens of other parameters in lightning trainer, why should we add here `precision` parameter but not all of the others? So far I've just tried to be minimalistic with this default config. We could also keep here all of the trainer parameters, but those often change in each new release so maintaining such config becomes bothersome